src/component/plotly-dash-preview/parse.js

```diff
@@ -31,6 +32,19 @@ function parse (body, opts, sendToRenderer) {
   result.loadingSelector = body.loading_selector
   result.timeOut = body.timeout
 
+  if (cst.sizeMapping[result.pdfOptions.pageSize]) {
+    result.browserSize = cst.sizeMapping[result.pdfOptions.pageSize]
+  } else if (body.pageSize && body.pageSize.width && body.pageSize.height) {
+    result.browserSize = {
+      width: body.pageSize.width * cst.pixelsInMicron,
+      height: body.pageSize.height * cst.pixelsInMicron
+    }
+  } else {
+    return errorOut(400, 'pageSize must either be A3, A4, A5, Legal, Letter, ' +
+                         'Tabloid or an Object containing height and width ' +
+                         'in microns.')
```

Comment:
:cow2: I really see indents like this in JS. That seems very pythonic. I would write this as:

```js
errorOut(
  400,
  'pageSize must either be A3, A4, A5, Legal, Letter, ' +
  'Tabloid or an Object containing height and width ' +
  'in microns.'
)
```